spinnaker_components/common/input_filtering.c

```diff
@@ -238,42 +238,54 @@ void input_filtering_get_routes(if_collection_t *filters,
                                 if_routing_table_t *routing_table,
                                 uint32_t *routes)
 {
-  // Copy in the number of routing entries
-  routing_table->n_routes = routes[0];
-  routes++;  // Advance the pointer to the first entry
-  debug("Loading %d filter routes\n", filters->n_routes);
+  filter_arg_t args[] = {{filters, routes}};
```

Comment:
Nicer syntax is, I believe this:
``filter_arg_t args[] = {.filter=filters, .data=routes};``